Secret-agency-web/src/main/react/secret-agency/src/registerServiceWorker.ts

```diff
@@ -0,0 +1,123 @@
+// tslint:disable:no-console
```

Comment:
Netuším, k čemu to je. Mohl bys to v krátkosti vystvětlit?

Reply:
v podstate na cachovani, aby to lepe fungovalo kdyz userovi spadne net atp., pro nase ucely asi k nicemu, je to tam default kdyz vytvoris react-app